code/AuditHook.php

```diff
@@ -122,12 +146,12 @@ public static function handle_manipulation($manipulation)
             }
 
             // log PermissionRole being added to a Group
-            if ($table == 'Group_Roles') {
-                $role = \PermissionRole::get()->byId($details['fields']['PermissionRoleID']);
-                $group = \Group::get()->byId($details['fields']['GroupID']);
+            if ($table === 'Group_Roles') {
```

Comment:
I know it's core so should be good, but would this not be better to be using `DataObjectSchema`? Technically it is a config var, and thus could be altered.

Reply:
I'm not sure we can, this is a pivot table rather than a physical model

Reply:
Parent table plus relation name - we can